app/org/sagebionetworks/bridge/models/AndroidAppSiteAssociation.java

```diff
@@ -0,0 +1,32 @@
+package org.sagebionetworks.bridge.models;
+
+import org.sagebionetworks.bridge.models.studies.AndroidAppLink;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * A model of the JSON we produce to associate app links on Android with our server. The JSON 
+ * that is returned is an array of these objects, one for every study that has defined app 
+ * links.
+ * 
+ * @see https://developer.android.com/training/app-links/verify-site-associations.html
+ */
+public final class AndroidAppSiteAssociation {
+    public static final String RELATION = "delegate_permission/common.handle_all_urls";
+    private static final String[] RELATION_LIST = new String[] {RELATION};
+    
+    private final AndroidAppLink target;
+    
+    @JsonCreator
+    public AndroidAppSiteAssociation(@JsonProperty("target") AndroidAppLink target) {
+        this.target = target;
+    }
```

Comment:
Do we need annotations on the constructor if we never deserialize this? (Here and in AndroidAppLink and AppleAppLink.)